components/docsMain/tocOverflowButton.tsx

```diff
@@ -0,0 +1,52 @@
+import Link from 'next/link';
+import { useState } from 'react';
+import { MdMenu } from 'react-icons/md';
+
+
+const TocOverflow = ({ tocData }) => {
+  return (
+    <div className="absolute z-10 bg-white mt-4 rounded-lg w-full p-6 shadow-lg animate-fade-down animate-duration-300 max-h-96 overflow-y-scroll">
+      {tocData.tocData.map((item, index) => {
+        const textIndentation =
+          item.type === 'h3' ? 'ml-4' : item.type === 'h4' ? 'ml-8' : '';
+        
+
+        const linkHref = `#${item.text.replace(/\s+/g, '-').toLowerCase()}`;
```

Comment:
be weary there is a utility function that does this same and is consistent across the site

Reply:
APOC - this works, however bugs always arise with these types of things so we should watch out